features/pages/flaw_detail_page.py

```diff
@@ -186,6 +189,17 @@ def set_input_field(self, field, value):
         self.driver.execute_script("arguments[0].value = '';", field_input)
         field_input.set_text(value)
 
+    def get_input_value(self, field):
+        field_value = getattr(self, field + 'Value')
+        self.driver.execute_script("arguments[0].scrollIntoView(true);", field_value)
+        return field_value.get_text()
+
+    def clear_input_field(self, field):
+        field_btn = field + 'EditBtn'
+        self.click_btn(field_btn)
+        field_input = getattr(self, field + 'Input')
+        field_input.send_keys(Keys.CONTROL + 'a', Keys.BACKSPACE)
```

Comment:
Why not use `execute_script("arguments[0].value = '';")`, not working?

Reply:
I also met the same issue. 
After using execute_script, add time.sleep(2), it works.